server/config.py

```diff
@@ -0,0 +1,4 @@
+SQLALCHEMY_DATABASE_URI = "postgresql://localhost"
```

Comment:
It's not uncommon to store configuration in `.py` files. Plus, these Flask docs seem to do it too https://exploreflask.com/configuration.html

Yeah, we can have a template when the environment grows more complex? Right now I really don't know if we'll use this extensively later.